manual/opt.tex

```diff
@@ -153,9 +153,108 @@ \subsubsection{quartic}
 \end{lstlisting}
 
 \subsubsection{adaptive}
-%To activate this optimizer, add ``-D BUILD\_LMYENGINE\_INTERFACE=1'' in the CMake command line.
 
-Manual is needed here.
+The default setting of the adaptive optimizer is to construct the linear method Hamiltonian and overlap matrices explicitly and add different shifts to the Hamiltonian matrix 
+as ``stablizers''. The generalized eigenvalue problem is solved for each shifted Hamiltonian and updates to wave function parameters are obtainted. Then a correlated sampling 
```

Comment:
It might read better as:
The generalized eigenvalue problem is solved for each shift to obtain updates to the wave function parameters.